sycl/include/sycl/kernel_bundle.hpp

```diff
@@ -201,6 +201,21 @@ class __SYCL_EXPORT kernel_bundle_plain {
     return ext_oneapi_get_kernel(detail::string_view{name});
   }
 
+  bool ext_oneapi_has_device_global(const std::string &name,
+                                    const device &dev) {
+    return ext_oneapi_has_device_global(detail::string_view{name}, dev);
```

Comment:
Why do we need the indirection via `string_view`?

Reply:
I followed the design for `ext_oneapi_has_kernel`, which underwent ABI neutralisation (cf. https://github.com/intel/llvm/pull/13447).